### PR TITLE
Fix plugin search

### PR DESCRIPTION
**Pull request by eslickj**
This fixes issue #462 

Added try/except and logging for finding future problems.   The problem was a 0x90 in ACOSSO.py.  I don't know how it got there, but I changed the plugin search code to read the files with UTF-8 encoding.  That fixed it.  That character must have gotten in recently somehow.

### CODE REVIEW

File: foqus_lib/framework/surrogate/ACOSSO.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 """ #FOQUS_SURROGATE_PLUGIN ACOSSO.py
 
-Surrogate plugins need to have FOQUS_SURROGATE_PLUGIN in the first
-150 characters of text.  They also need to hav a .py extention and
-inherit the surrogate class.
+Surrogate plugins need to have FOQUS_SURROGATE_PLUGIN in the first 150
+characters of text.  They also need to hav a .py extention and inherit the
```
Comment (perrenyang) [inline foqus_lib/framework/surrogate/ACOSSO.py]: I'd like to take this chance to correct "extention" to "extension"

**Comment by perrenyang** [comment]
Awesome, looks like it works (although now AppVeyor is running into some different exception while attempting to test...)

**Comment by eslickj** [comment]
I fixed the spelling error too, which due to copy and paste appeared in a lot of places.

**Comment by eslickj** [comment]
The exception stopping appveyor doen't stop FOQUS from running, but I'll try to track it down.

**Pull request merged**